tools/esmodules-export-gen.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
```

Comment:
Just a proposal/idea, maybe add a shebang for this file? It's up to you, so if you feel there is no need for it, just skip this suggestion.
```suggestion
#!/usr/bin/env node
'use strict';
```

Reply:
Done.